packager/moduleNames.js

```diff
@@ -209,6 +209,29 @@ module.exports = [
     'node_modules/@react-native-community/async-storage/lib/AsyncStorage.js',
     'node_modules/@react-native-community/async-storage/lib/index.js',
     'node_modules/@react-native-community/netinfo/js/index.js',
+    'node_modules/@react-navigation/core/lib/module/NavigationActions.js',
```

Comment:
Please remove the `@react-navigation` references here

Reply:
those are only needed for the extension, so no need to include them in the main bundle